src/todo/todo.controller.ts

```diff
@@ -0,0 +1,39 @@
+import { Controller, Get, Post, Put, Delete, Patch, Param, Body, ParseIntPipe } from '@nestjs/common';
+import { TodoDto } from './dto/todo.dto';
+import { Todo } from './todo.entity';
+import { TodoService } from './todo.service';
+
+@Controller('todo')
```

Comment:
I was not able to find test cases for these endpoints. Can you please add the required test cases ?

Reply:
Test cases have now been added to the `todo.controller.spec.ts` file